### PR TITLE
Added different level of logs throughout whole codebase.

### PR DESCRIPTION
**Pull request by Satendra-SR**
#338 

### Types of changes

- [ ] Bug fix (a change which fixes an issue)
- [x] New feature (a change which adds functionality)

### Short description of what this resolves:

**Why these change required? What problem does it solve?**
By adding different levels of logs to the whole code base, we can get logs to provide detailed information about what is happening in the code, making it easier to debug issues and identify the root cause of problems.
  Here we are implementing the following level of logs in the system.
- INFO
- WARN
- ERROR

### Checklist:

- [x] I have performed a self-review of my own code.
- [x] The code follows the style guidelines of this project.
- [x] The code changes are passing the CI checks
- [ ] I have documented my code wherever required.
- [ ] The changes requires a change to the documentation.
- [ ] I have updated the documentation based on the my changes.

### CODE REVIEW

File: api/services/contact_fields_mapping_service.py [hunk 1]
```diff
@@ -105,5 +122,10 @@ def update_mapped_fields(
                 if getattr(table_data, column_name) != mapped_table_column_value:
                     setattr(table_data, column_name, mapped_table_column_value)
                     db.session.commit()
+                    logger.info(
+                        f"Successfully updated {column_name} with value {mapped_table_column_value} for user with id {user_details.id}"
+                    )
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:127]: This is not required

File: api/services/program_sequence_service.py [hunk 2]
```diff
@@ -43,6 +45,7 @@ def get_program_sequence_id(self, jsonData):
 
             if not user_program_details:
                 # Need to log this
+                logger.error("No user program details found")
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:48]: Can we print user_id as well

File: api/services/prompt_service.py [hunk 3]
```diff
@@ -10,24 +12,41 @@ def __init__(self):
...
+            if "flow_run_details" in jsonData:
+                flow_run_uuid = helpers.fetch_by_key(
+                    "uuid", jsonData["flow_run_details"]
+                )
+            call_log_details = models.CallLog.query.get_by_flow_run_uuid(flow_run_uuid)
+            self.call_log_id = call_log_details.id
+            self.content_version_id = call_log_details.content_version_id
+            logger.info(
+                f" user_phone: {self.user_phone}, call_log_id: {self.call_log_id}, content_version_id: {self.content_version_id}"
+            )
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:30]: Not needed

File: api/services/prompt_service.py [hunk 4]
```diff
@@ -52,6 +78,12 @@ def handle_prompt_response(self, jsonData):
                     response_exists = self.if_exists(
                         ivr_prompt_response_details, prompt_name, prompt_response
                     )
+                    if response_exists:
+                        logger.info(
+                            "IVR prompt response exists for prompt '{}' and response '{}'".format(
+                                prompt_name, prompt_response
+                            )
+                        )
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:86]: Not nededed

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: api/services/contact_fields_mapping_service.py (hunk 1 above)
Comment (Satendra-SR) [reply api/services/contact_fields_mapping_service.py]: Removed

File: api/services/program_sequence_service.py (hunk 2 above)
Comment (Satendra-SR) [reply api/services/program_sequence_service.py]: Added

File: api/services/prompt_service.py (hunk 3 above)
Comment (Satendra-SR) [reply api/services/prompt_service.py]: Removed

File: api/services/prompt_service.py (hunk 4 above)
Comment (Satendra-SR) [reply api/services/prompt_service.py]: Removed

File: api/helpers/db_helper.py [hunk 5]
```diff
@@ -1,15 +1,23 @@
...
+            partner_system_phone = (
+                models.PartnerSystemPhone.query.get_by_system_phone_id(
+                    system_phone_details.id
+                )
+            )
+            if partner_system_phone:
+                return partner_system_phone.partner_id
+    except Exception as e:
+        logger.error(
+            f"Error while fetching partner id by system phone {system_phone}: {e}"
```
Comment (GauravGusain98) [inline api/helpers/db_helper.py:19]:
```suggestion
            f"Error while fetching partner id by system phone {system_phone}. Error message: {e}"
```

File: api/helpers/db_helper.py [hunk 6]
```diff
@@ -18,8 +26,10 @@ def save(data):
         db.session.add(data)
         db.session.commit()
     except Exception as e:
-        print("Error: " + str(e))
-        print(traceback.format_exc())
+        logger.error(
+            f"Error occurred while commiting the data in the database. Error Message:  {e}"
```
Comment (GauravGusain98) [inline api/helpers/db_helper.py:30]:
```suggestion
            f"Error occurred while committing the data in the database. Error message:  {e}"
```

File: api/helpers/prompt_helper.py [hunk 7]
```diff
@@ -1,42 +1,57 @@
...
 
 
 def split_prompt_by_hyphen(data):
-    split_prompt = data.split("-")
-    return split_prompt
+    try:
+        split_prompt = data.split("-")
+        return split_prompt
+    except Exception as e:
+        logger.error(f"Error while splitting prompt by hyphen: {e}")
```
Comment (GauravGusain98) [inline api/helpers/prompt_helper.py:10]:
```suggestion
        logger.error(f"Error while splitting prompt '{data}' by hyphen. Error message: {e}")
```

File: api/helpers/prompt_helper.py [hunk 8]
```diff
@@ -1,42 +1,57 @@
...
 
 
 def split_prompt_by_underscore(data):
-    program_sub_prompt = data.split("_")
-    return program_sub_prompt
+    try:
+        program_sub_prompt = data.split("_")
+        return program_sub_prompt
+    except Exception as e:
+        logger.error(f"Error while splitting prompt by underscore: {e}")
```
Comment (GauravGusain98) [inline api/helpers/prompt_helper.py:18]:
```suggestion
        logger.error(f"Error while splitting prompt '{data}' by underscore. Error message: {e}")
```

File: api/helpers/prompt_helper.py [hunk 9]
```diff
@@ -1,42 +1,57 @@
...
+                split_prompt_by_underscore = helpers.split_prompt_by_underscore(
+                    split_prompt_by_hyphen[-1]
+                )
+                return (
+                    split_prompt_by_underscore[1]
+                    if len(split_prompt_by_underscore) > 1
+                    else None
+                )
+    except Exception as e:
+        logger.error(f"Error while getting program prompt id: {e}")
```
Comment (GauravGusain98) [inline api/helpers/prompt_helper.py:44]: This try and except is not needed.

File: api/helpers/prompt_helper.py [hunk 10]
```diff
@@ -1,42 +1,57 @@
...
-
-    return column_type.fetchone()[0]
+    try:
+        get_column_type_query = f"SELECT DATA_TYPE FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = '{table_name}' AND COLUMN_NAME  = '{column_name}'"
+        column_type = db.session.execute(get_column_type_query)
+
+        return column_type.fetchone()[0]
+    except Exception as e:
+        logger.error(
+            f"An error occurred while getting data type for column {column_name} in table {table_name}: {e}"
```
Comment (GauravGusain98) [inline api/helpers/prompt_helper.py:56]:
```suggestion
            f"An error occurred while getting data type for column {column_name} in table {table_name}. Error message: {e}"
```

File: api/models/contact_fields_mapping.py [hunk 11]
```diff
@@ -10,16 +11,21 @@ def get_by_field_name(self, field_name, field_value):
                 ContactFieldsMapping.expected_field_value == field_value,
             ).all()
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(f"Exception occurred while getting field name: {e}")
```
Comment (GauravGusain98) [inline api/models/contact_fields_mapping.py:14]:
```suggestion
            logger.error(f"Exception occurred while fetching contact field mapping using field name: {field_name} and value: {field_value}. Error message: {e}")
```

File: api/models/contact_fields_mapping.py [hunk 12]
```diff
@@ -10,16 +11,21 @@ def get_by_field_name(self, field_name, field_value):
...
 
     def get_by_group_name(self, contact_groups):
         try:
             return self.filter(ContactFieldsMapping.field_name == contact_groups).all()
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(f"Exception occurred while getting group name: {e}")
```
Comment (GauravGusain98) [inline api/models/contact_fields_mapping.py:20]:
```suggestion
            logger.error(f"Exception occurred while fetching contact field mapping using group name {contact_groups}. Erorr message: {e}")
```

File: api/models/user_program.py [hunk 13]
```diff
@@ -37,26 +38,39 @@ def update(self, user_program_details, data):
...
+                    UserProgram.user_id == user_id,
+                    UserProgram.status
+                    == models.UserProgram.UserProgramStatus.IN_PROGRESS,
+                )
+                .order_by(UserProgram.id.desc())
+                .first()
+            )
+        except Exception as e:
+            logger.error(
+                f"Failed to fetch the latest active user program for user_id {user_id}: {e}"
```
Comment (GauravGusain98) [inline api/models/user_program.py:58]:
```suggestion
                f"Failed to fetch the latest active user program for user_id {user_id}. Error message: {e}"
```

File: api/models/user_program.py [hunk 14]
```diff
@@ -37,26 +38,39 @@ def update(self, user_program_details, data):
 
             db.session.commit()
         except Exception as e:
-            print(f"Failed to update user program details: {e}")
+            logger.error(
+                f"Failed to update user program details for user_program_id {user_program_details.id}: {e}"
```
Comment (GauravGusain98) [inline api/models/user_program.py:42]:
```suggestion
                f"Failed to update user program details for user_program_id {user_program_details.id}. Error message: {e}"
```

File: api/models/user_program.py [hunk 15]
```diff
@@ -37,26 +38,39 @@ def update(self, user_program_details, data):
...
+            return (
+                self.filter(
+                    UserProgram.user_id == user_id,
+                )
+                .order_by(UserProgram.id.desc())
+                .first()
+            )
+        except Exception as e:
+            logger.error(
+                f"Error occurred while fetching latest user program for user_id: {user_id}. Error: {e}"
```
Comment (GauravGusain98) [inline api/models/user_program.py:72]:
```suggestion
                f"Error occurred while fetching latest user program for user_id: {user_id}. Error message: {e}"
```

File: api/services/call_log_service.py [hunk 16]
```diff
@@ -37,14 +38,15 @@ def handle_flow_run_details(self, jsonData):
                     "created_on", jsonData["flow_run_details"]
                 )
 
-        except:
-            print("Failed to fetch flow run details")
+        except Exception as e:
+            logger.error(
+                f"Failed to fetch flow run details for user phone {self.user_phone}: {e}"
```
Comment (GauravGusain98) [inline api/services/call_log_service.py:43]:
```suggestion
                f"Failed to fetch flow run details for user phone {self.user_phone}. Error message: {e}"
```

File: api/services/call_log_service.py [hunk 17]
```diff
@@ -135,9 +136,9 @@ def create_call_logs(self, jsonData):
             )
             helpers.save(new_call_log)
             self.call_log = new_call_log
-        except:
+        except Exception as e:
             # Need to log this
-            return "Failed to create call log"
+            logger.error(f"Failed to create call log for {self.user_phone}: {e}")
```
Comment (GauravGusain98) [inline api/services/call_log_service.py:141]:
```suggestion
            logger.error(f"Failed to create call log for {self.user_phone}. Error message: {e}")
```

File: api/services/call_log_service.py [hunk 18]
```diff
@@ -154,9 +155,11 @@ def update_call_logs(self, data):
                     self.call_log.content_version_id = content_version_id
 
             db.session.commit()
-        except:
+        except Exception as e:
             # Need to log this
-            return "Failed to udpate call log"
+            logger.error(
+                f"Failed to update call log for {self.user_phone}: {e}", exc_info=True
```
Comment (GauravGusain98) [inline api/services/call_log_service.py:161]: What the use of `exc_info=True`. And why we have not used it on other places?

File: api/services/contact_fields_mapping_service.py [hunk 19]
```diff
@@ -27,7 +29,9 @@ def handle_contact_fields_data(self, jsonData):
                             contact_field_mapping, user_details, field_value
                         )
         except Exception as e:
-            print(f"Error occurred while handling custom field mapping. Exception: {e}")
+            logger.error(
+                f"Error occurred while handling custom field mapping for {self.user_phone}. Exception: {e}"
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:33]: Lets make the language consistent
```suggestion
                f"Error occurred while handling custom field mapping for {self.user_phone}. Error message: {e}"
```

File: api/services/contact_fields_mapping_service.py [hunk 20]
```diff
@@ -45,7 +49,9 @@ def handle_contact_groups_data(self, jsonData):
                             custom_group_mapping, user_details
                         )
         except Exception as e:
-            print(f"Error occurred while handling custom field mapping. Exception: {e}")
+            logger.error(
+                f"Error occurred while handling custom field mapping for {self.user_phone}. Exception: {e}"
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:53]:
```suggestion
                f"Error occurred while handling custom field mapping for {self.user_phone}. Error message: {e}"
```

File: api/services/contact_fields_mapping_service.py [hunk 21]
```diff
@@ -75,7 +81,9 @@ def process_contact_fields_data(
                 user_details,
             )
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(
+                f"Exception occurred while processing contact field data for {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:85]:
```suggestion
                f"Exception occurred while processing contact field data for {self.user_phone}. Error message: {e}"
```

File: api/services/contact_fields_mapping_service.py [hunk 22]
```diff
@@ -94,7 +102,9 @@ def process_contact_groups_data(self, user_contact_group_details, user_details):
                     user_details,
                 )
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(
+                f"Exception occurred while processsing contact group data for {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:106]:
```suggestion
                f"Exception occurred while processing contact group data for {self.user_phone}. Error message: {e}"
```

File: api/services/contact_fields_mapping_service.py [hunk 23]
```diff
@@ -106,4 +116,6 @@ def update_mapped_fields(
                     setattr(table_data, column_name, mapped_table_column_value)
                     db.session.commit()
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(
+                f"Exception occurred while updating mapped fields for {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/contact_fields_mapping_service.py:120]:
```suggestion
                f"Exception occurred while updating mapped fields for {self.user_phone}. The column name is {column_name} and the value is {mapped_table_column_value}. Error: {e}"
```

File: api/services/program_sequence_service.py [hunk 24]
```diff
@@ -43,6 +41,9 @@ def get_program_sequence_id(self, jsonData):
 
             if not user_program_details:
                 # Need to log this
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:43]: Remove this comment

File: api/services/program_sequence_service.py [hunk 25]
```diff
@@ -59,4 +60,7 @@ def get_program_sequence_id(self, jsonData):
 
             return program_sequence_id
         except Exception as e:
-            print(f"Exception occured:{e}")
+            logger.error(
+                f"Exception occured while getting program sequence id for {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/program_sequence_service.py:64]:
```suggestion
                f"Exception occured while getting program sequence id for {self.user_phone}. Error message: {e}"
```

File: api/services/prompt_service.py [hunk 26]
```diff
@@ -122,43 +150,50 @@ def handle_prompt_mapping(
                 self.process_mapped_fields(
                     user_details, ivr_prompt_mapping_data, prompt_response_value
                 )
-        except:
-            print("Exception occured")
+        except Exception as e:
+            logger.error(
+                f"Exception occurred while handling prompt mapping for user phone {self.user_phone}: {e}"
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:155]:
```suggestion
                f"Exception occurred while handling prompt mapping for user phone {self.user_phone}. Error message {e}"
```

File: api/services/prompt_service.py [hunk 27]
```diff
@@ -122,43 +150,50 @@ def handle_prompt_mapping(
...
+
+                    self.update_mapped_fields(
+                        class_object,
+                        user_details,
+                        column_name,
+                        prompt_response_value,
+                    )
+        except Exception as e:
+            logger.error(
+                f"Failed to process mapped fields for user phone {self.user_phone}. Error:{e}"
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:195]:
```suggestion
                f"Failed to process mapped fields for prompt value '{prompt_response_value}' and for user phone {self.user_phone}. Error message:{e}"
```

File: api/services/prompt_service.py [hunk 28]
```diff
@@ -169,7 +204,9 @@ def update_mapped_fields(
                 setattr(class_object_data, column_name, prompt_response_value)
                 db.session.commit()
         except Exception as e:
-            print(f"Exception occurred: {e}")
+            logger.error(
+                f"Exception occurred while updating mapped fields for {self.user_phone}: {e}"
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:208]:
```suggestion
                f"Exception occurred while updating mapped fields for prompt value = {prompt_response_value} and for {self.user_phone}. Error message {e}"
```

File: api/services/prompt_service.py [hunk 29]
```diff
@@ -178,7 +215,10 @@ def sanitize_keypress(self, data):
                 return_keypress_value = int(keypress)
             else:
                 return_keypress_value = -2
-        except:
+        except Exception as e:
+            logger.error(
+                f"Keypress value by {self.user_phone} is not an integer. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/prompt_service.py:220]:
```suggestion
                f"Keypress value by {self.user_phone} is not an integer. Error message: {e}"
```

File: api/services/registration_service.py [hunk 30]
```diff
@@ -62,7 +63,9 @@ def handle_registration(self, jsonData):
                     models.UserProgram.query.create(self.user_id, user_program_data)
 
         except Exception as e:
-            print(f"Failed to handle registration: {e}")
+            logger.error(
+                f"Failed to handle registration for {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/registration_service.py:67]:
```suggestion
                f"Failed to handle registration for {self.user_phone}. Error messager: {e}"
```

File: api/services/registration_service.py [hunk 31]
```diff
@@ -91,7 +94,12 @@ def register(self, jsonData):
                 == models.Registration.RegistrationStatus.COMPLETE
                 else None,
             )
-            helpers.save(registrant)
+            try:
+                helpers.save(registrant)
+            except Exception as e:
+                logger.error(
+                    f"Failed to save registration data for user with phone {self.user_phone}. Error: {e}"
```
Comment (GauravGusain98) [inline api/services/registration_service.py:101]:
```suggestion
                    f"Failed to save registration data for user with phone {self.user_phone}. Error message: {e}"
```

File: api/services/registration_service.py [hunk 32]
```diff
@@ -120,7 +128,12 @@ def update_registration(self, registration, jsonData):
         if self.user_id:
             registration.user_id = self.user_id
             registration.has_received_callback = True
-            db.session.commit()
+            try:
+                db.session.commit()
+            except Exception as e:
+                logger.error(
+                    f"Failed to update registration for {self.user_phone}: {e}"
```
Comment (GauravGusain98) [inline api/services/registration_service.py:135]:
```suggestion
                    f"Failed to update registration for {self.user_phone}. Error message: {e}"
```

File: api/services/transaction_log_service.py [hunk 33]
```diff
@@ -1,32 +1,42 @@
...
-        helpers.save(new_webhook_log)
-        return new_webhook_log
+        try:
+            new_webhook_log = models.WebhookTransactionLog(
+                payload=json.dumps(jsonData), processed=False
+            )
+            helpers.save(new_webhook_log)
+            return new_webhook_log
+        except Exception as e:
+            logger.error(f"Error while creating new webhook log: {e}")
```
Comment (GauravGusain98) [inline api/services/transaction_log_service.py:16]:
```suggestion
            logger.error(f"Error while creating new webhook log. Webhook: {jsonData}. Error message: {e}")
```

File: api/services/transaction_log_service.py [hunk 34]
```diff
@@ -1,32 +1,42 @@
...
 
     def mark_webhook_log_as_processed(self, webhook_log):
-        webhook_log.processed = True
-        helpers.save(webhook_log)
+        try:
+            webhook_log.processed = True
+            helpers.save(webhook_log)
+        except Exception as e:
+            logger.error(f"Error while marking webhook log as processed: {e}")
```
Comment (GauravGusain98) [inline api/services/transaction_log_service.py:23]:
```suggestion
            logger.error(f"Error while marking webhook log as processed. Error message: {e}")
```

File: api/services/transaction_log_service.py [hunk 35]
```diff
@@ -1,32 +1,42 @@
...
             )
-            .order_by(models.WebhookTransactionLog.created_on)
-            .limit(app.config["RETRY_LOGS_BATCH_LIMIT"])
-            .all()
-        )
 
-        return failed_webhook_transaction_logs
+            return failed_webhook_transaction_logs
+        except Exception as e:
+            logger.error(f"Error while retrieving failed webhook transaction logs: {e}")
```
Comment (GauravGusain98) [inline api/services/transaction_log_service.py:42]:
```suggestion
            logger.error(f"Error while retrieving failed webhook transaction logs. Error message: {e}")
```

File: main.py [hunk 36]
```diff
@@ -27,8 +24,10 @@ def webhook(request):
                 processed = handle_payload(jsonData)
 
                 if processed is False:
+                    logger.error("[ERROR] Error processing the payload")
```
Comment (GauravGusain98) [inline main.py:27]:
```suggestion
                    logger.error("Error processing the payload: {josnData}")
```

File: main.py [hunk 37]
```diff
@@ -27,8 +24,10 @@ def webhook(request):
...
                     return jsonify(message="Something went wrong!"), 400
                 elif processed == -1:
+                    logger.warning("[ERROR] Contact not found in the payload")
```
Comment (GauravGusain98) [inline main.py:30]: Not use this if we are not using it on other logs
```suggestion
                    logger.warning("Contact not found in the payload")
```

File: main.py [hunk 38]
```diff
@@ -38,12 +37,13 @@ def webhook(request):
...
             return jsonify(message="Success"), 200
         else:
+            logger.warning("[WARN] Received a GET request instead of POST")
             return (
                 jsonify(message="Currently, the system do not accept a GET request"),
                 405,
             )
     except Exception as e:
-        print(e)
+        logger.error("[ERROR] An unexpected error occurred: %s" % e)
```
Comment (GauravGusain98) [inline main.py:46]:
```suggestion
        logger.error(f"An unexpected error occurred. Error message: {e}" )
```

File: main.py [hunk 39]
```diff
@@ -63,6 +63,7 @@ def retry_failed_webhook(transaction_log_service):
 
         log.processed = True
         db_helper.save(log)
+        logger.info("[INFO] Successfully processed the failed log")
```
Comment (GauravGusain98) [inline main.py:66]: No needed

File: main.py [hunk 40]
```diff
@@ -116,9 +117,12 @@ def handle_payload(jsonData, is_retry_payload=False):
             if jsonData.get("flow_category") == "dry_flow" and not is_retry_payload:
                 handle_contact_fields_and_groups(jsonData)
         else:
+            logger.error("[ERROR] No 'contact' key found in the input JSON data.")
```
Comment (GauravGusain98) [inline main.py:120]:
```suggestion
            logger.error("No 'contact' key found in the input JSON data. {josnData}")
```

File: main.py [hunk 41]
```diff
@@ -116,9 +117,12 @@ def handle_payload(jsonData, is_retry_payload=False):
...
             return -1
-    except:
+    except Exception as e:
+        logger.error(f"[ERROR] Exception Occured while handeling payload: {e}")
```
Comment (GauravGusain98) [inline main.py:123]:
```suggestion
        logger.error(f"Exception occurred while handling payload: {e}")
```

File: main.py [hunk 42]
```diff
@@ -116,9 +117,12 @@ def handle_payload(jsonData, is_retry_payload=False):
...
         return False
+    logger.info("[INFO] Payload processing completed successfully.")
```
Comment (GauravGusain98) [inline main.py:125]: Not needed

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

File: api/helpers/prompt_helper.py (hunk 9 above)
Comment (Satendra-SR) [reply api/helpers/prompt_helper.py]: Removed

File: main.py (hunk 39 above)
Comment (Satendra-SR) [reply main.py]: Removed

File: main.py (hunk 42 above)
Comment (Satendra-SR) [reply main.py]: Removed

**Review by GauravGusain98 — APPROVED** [review]

**Pull request merged**